cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/ScalarFunctionsTest.scala

```diff
@@ -117,13 +118,18 @@ class ScalarFunctionsTest extends DocumentingTest {
       }
     }
     section("id()", "functions-id") {
-      p("The function `id()` returns the id of a relationship or node.")
+      p("The function `id()` returns an identifier, the function can be utilized for a relationship or a node.")
```

Comment:
Either 2 sentences: "The function `id()` returns an identifier. The function can be utilized for a relationship or a node." or a ";"
"The function `id()` returns an identifier; the function can be utilized for a relationship or a node."